wordy.php

```diff
@@ -4,10 +4,14 @@
 // This is only a SKELETON file for the "Hello World" exercise.
 // It's been provided as a convenience to get you started writing code faster.
 //
-
-function calculate($question = "")
-{
-    //
-    // YOUR CODE GOES HERE
-    //
+ function HammingDistance($x, $y) {
```

Comment:
You solve the **wrong** challenge :laughing: !!!
That's **Hamming** solution, here it's **Wordy** exercise..